samples/tails/README.md

```diff
@@ -0,0 +1,5 @@
+<h1 align="center"><b>Test tails file server</b></h1>
+
+This is a very simple server that is used to host tails files during for tests suites where revocable credentials are issued and verified.
```

Comment:
Couldn't we use file paths in the revocation registries for tests?

Reply:
Actually, in tests we are using the `InMemoryTailsFileService`, so this sample tails server is only kept as a reference (not sure if it's the best place to store it though). The README is a bit misleading: I'll update it